misc/schema/xrc_schema.rnc

```diff
@@ -473,8 +473,8 @@ t_bitmap     = t_url?,
 t_font       = (
                    [xrc:p="o"] element size         {_, t_float }* &
                    [xrc:p="o"] element style        {_, ("normal" | "italic" | "slant") }* &
-                   [xrc:p="o"] element weight       {_, ("normal" | "thin" | "extralight" | "light" | 
-                                                         "medium" | "semibold" | "bold" | "extrabold" | 
+                   [xrc:p="o"] element weight       {_, ("normal" | "thin" | "extralight" | "light" |
```

Comment:
This is very minor, but please avoid including whitespace-only changes in the commits with other, actually significant changes (this probably means configuring your editor to avoid stripping whitespace automatically from wx files).

Reply:
Ack, didn't mean to do that. I switched my editor to both show me when whitespace changed, and specific to this repository to not trim trailing whitespace. Shouldn't happen again...